src/main/java/de/tum/in/www1/artemis/domain/TextAssessmentConflict.java

```diff
@@ -0,0 +1,99 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+import javax.persistence.*;
+
+import de.tum.in.www1.artemis.domain.enumeration.TextAssessmentConflictType;
+
+/**
+ * Represents the conflicts between two feedback of a text exercise.
+ */
+@Entity
+@Table(name = "text_assessment_conflict")
+public class TextAssessmentConflict implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "conflict", nullable = false)
+    private Boolean conflict;
+
+    @Column(name = "created_at")
+    private ZonedDateTime created_at;
+
+    @Column(name = "solved_at")
+    private ZonedDateTime solved_at;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "type", nullable = false)
+    private TextAssessmentConflictType type;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "first_feedback_id", referencedColumnName = "id")
+    private Feedback firstFeedback;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "second_feedback_id", referencedColumnName = "id")
+    private Feedback secondFeedback;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public Boolean getConflict() {
+        return conflict;
+    }
+
+    public void setConflict(Boolean conflict) {
+        this.conflict = conflict;
+    }
+
+    public ZonedDateTime getCreated_at() {
+        return created_at;
+    }
+
+    public void setCreated_at(ZonedDateTime created_at) {
+        this.created_at = created_at;
+    }
+
+    public ZonedDateTime getSolved_at() {
```

Comment:
Codacy found an issue: [The instance method name 'getSolved_at' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968856)